coralproject/talk
# [CORL-2552] Update Sentry to latest (6.19.7)

## What does this PR do?

Update Sentry to latest (6.19.7).

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags? 

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- build the app
- see it is not broke
- type checking will ensure that Sentry API is compatible via TS 
 
## How do we deploy this PR?

No special considerations.
